Sources/Errors.swift

```diff
@@ -37,6 +37,15 @@ public struct TemplateSyntaxError : Error, Equatable, CustomStringConvertible {
   public init(_ description: String) {
     self.init(reason: description)
   }
+    
+  public static func ==(lhs: TemplateSyntaxError, rhs: TemplateSyntaxError) -> Bool {
+    return lhs.reason == rhs.reason &&
+    lhs.description == rhs.description &&
+    lhs.token == rhs.token &&
+    lhs.stackTrace == rhs.stackTrace &&
+    lhs.templateName == rhs.templateName &&
+    lhs.allTokens == rhs.allTokens
```

Comment:
`allTokens` check is not needed as it is computed property. Also lines starting with 43 should be indented on one position.

Reply:
I'd also wrap this whole `==` operator in a `#if swift` check.